.github/workflows/docs/conf.py

```diff
@@ -44,7 +42,7 @@
 # The following code is for resolving broken hyperlinks in the doc.
```

Comment:
This file can actually be deleted along with the entire `.github/workflows/docs` directory. 

I can do this in a follow up PR if preferred.

Reply:
Done